src/Dfe.PlanTech.Application/Persistence/Interfaces/IDbContext.cs

```diff
@@ -2,6 +2,8 @@ namespace Dfe.PlanTech.Application.Persistence.Interfaces;
 
 public interface IDbContext
 {
+    public Task<T?> FirstOrDefaultCachedAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default);
+    public Task<List<T>> ToListCachedAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default);
```

Comment:
Just making sure I follow - is the reason that you've done this so that the service bus and related code can use the non-cached methods, but all usage within app code uses the cached methods?

Reply:
Yup